Swift Vocabulary/Swift Vocabulary/DefinitionViewController.swift

```diff
@@ -10,23 +10,22 @@ import UIKit
 
 class DefinitionViewController: UIViewController {
 
+    var vocabWord: VocabularyWord?
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        updateViews()
         // Do any additional setup after loading the view.
```

Comment:
Comments should be removed before submission (mainly on Sprint Challenges - homework projects aren't as strict)